post_processing_scripts/VaryTempWithHeight.py

```diff
@@ -89,23 +98,24 @@ def execute(self, data):
 
                 # If we've found our line
                 if match is not None:
+                    output_line += ';TYPE:CUSTOM\n'
```

Comment:
This doesn't need to be on every line, this should only show up when we print our custom temp change line